packages/components/src/navigator/navigator-screen/component.tsx

```diff
@@ -100,11 +100,13 @@ function UnconnectedNavigatorScreen(
 		// - when the screen becomes visible
 		// - if the wrapper ref has been assigned
 		// - if focus hasn't already been restored for the current location
+		// - if the `skipFocus` option is set to `true`. This is useful when we trigger the navigation outside of NavigatorScreen.
```

Comment:
I believe the logic in the comment is flipped? We try to restore focus if the `skipFocus` option is _not _ set to `true`